controllers/auth.js

```diff
@@ -0,0 +1,38 @@
+ 
+import usersInfo from '../models/usersInfo';
+// import Joi from 'joi';
+// import jwt from 'jsonwebtoken';
+// import bcrypt from 'bcrypt';
+
+ class AuthController{
+ 	static signup(request, response){
+			usersInfo.push(request.user);
+
+			response.status(201).json({
```

Comment:
@cynmumbua , you are repeating res.status().json() . everywhere in your code.. nextime try and have an helper function that will do that for you